helm/prometheus-meta-operator/templates/ingress-oauth.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "resource.default.name" . }}
+  namespace: "monitoring"
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  {{- if .Values.Installation.V1.Monitoring.Prometheus.Letsencrypt }}
+  annotations:
+    kubernetes.io/tls-acme: "true"
```

Comment:
Coud we not actually write it as kubernetes.io/tls-acme: {{ .Values.Installation.V1.Monitoring.Prometheus.Letsencrypt }} instead and remove the if?